tests/integration_tests/functional/test_balloon.py

```diff
@@ -65,7 +65,10 @@ def make_guest_dirty_memory(ssh_connection, amount_mib=32):
 
     lower_ssh_oom_chance(ssh_connection)
 
-    cmd = f"/usr/local/bin/fillmem {amount_mib}"
+    # Start fillmem in the background so that we return as soon as possible
+    # so that we avoid having the SSH connections hanging due to the OOM
+    # killer kicking in.
+    cmd = f"/usr/local/bin/fillmem {amount_mib} &"
```

Comment:
In fact I think the & is not actually taking effect. To really disassociate from ssh one needs to do sthg like: 

```
uvm.ssh.run("sleep 5  &>/dev/null &")
```

Reply:
Hmmm, it could be. This was just an effort to return from the SSH call as soon as possible. I assume this cuts down time, but if indeed fillmem forks it might not save much. Also, with the SSH timeout trick it might not matter. I don't have strong feelings, WDYT?

Reply:
I dropped it